src/capitalize.js

```diff
@@ -0,0 +1,3 @@
+export default function capitalize(name) {
+  return name.charAt(0).toUpperCase() + name.slice(1);
```

Comment:
I don’t think this will work reliably with some locales and Unicode names; https://github.com/tc39/ecma402/issues/294 discusses it in more detail - not sure if it matters for filenames tho.